lib/secure_headers/headers/content_security_policy.rb

```diff
@@ -147,6 +148,19 @@ def reject_all_values_if_none(source_list)
       end
     end
 
+    def normalize_uri_paths(source_list)
+      source_list.map do |source|
+        # Normalize domains ending in a single / as without omitting the slash accomplisheg the same.
+        # https://www.w3.org/TR/CSP3/#match-paths § 6.6.2.10 Step 2
+        if source.chomp("/").include?("/")
```

Comment:
One thing this might not be normalizing is a CSP directive like `https://www.example.com/` since the protocol includes `/`.

The quoted spec is about matching on the `path-part`, not the entire URL that is in the directive.

It would be good to address this, with unit tests as well.